composer.json

```diff
@@ -36,12 +36,14 @@
     "autoload": {
         "psr-4": {
             "AsyncAws\\CloudFormation\\": "src/Service/CloudFormation/src",
+            "AsyncAws\\CloudWatchLogs\\": "src/Service/CloudWatchLogs/src",
             "AsyncAws\\CodeDeploy\\": "src/Service/CodeDeploy/src",
             "AsyncAws\\DynamoDb\\": "src/Service/DynamoDb/src",
             "AsyncAws\\CodeGenerator\\": "src/CodeGenerator/src",
             "AsyncAws\\Core\\": "src/Core/src",
             "AsyncAws\\Flysystem\\S3\\": "src/Integration/Flysystem/S3/src",
             "AsyncAws\\Lambda\\": "src/Service/Lambda/src",
+            "AsyncAws\\MonologV2\\CloudWatch\\": "src/Integration/MonologV2/CloudWatch/src",
```

Comment:
We only provide 1 integration per application for all supported version. Use just `Monolog`